dependencies/graknlabs/repositories.bzl

```diff
@@ -29,10 +29,10 @@ def graknlabs_graql():
 def graknlabs_common():
     git_repository(
         name = "graknlabs_common",
-        remote = "https://github.com/graknlabs/common",
-        tag = "2.0.0-alpha-6" # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_common
+        remote = "https://github.com/lolski/common",
+        commit = "a2dafd47577cc63cee640ee331472f65722e7c70" # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_common
```

Comment:
Is there a `common` PR inbound?

Reply:
Yes, for adding `GraknClusterRunner`. I'm still tidying it up.